src/assets/i18n/es-ar.json

```diff
@@ -492,54 +492,54 @@
-  "{tasks, plural, =1 {# received task} other {# received tasks}} this week": "",
-  "{tasks, plural, =1 {# send task} other {# send tasks} }": "",
-  "{tasks, plural, =1 {# sent task} other {# sent tasks}} this week": "",
+  "{ n, plural, one {# snapshot} other {# snapshots} }": "{ n, plural, one {# snapshot} other {# snapshots} }",
+  "{cpuPercentage}% Avg. Usage": "Uso promedio del {cpuPercentage}%",
+  "{crontabDescription}, from {startHour} to {endHour}": "{crontabDescription}, desde {startHour} hasta {endHour}",
+  "{days, plural, =1 {# day} other {# days}}": "{days, plural, =1 {# day} other {# days}}",
+  "{email} via {server}": "{email} via {server}",
+  "{failedCount} of {allCount, plural, =1 {# task} other {# tasks}} failed": "{failedCount} of {allCount, plural, =1 {# task} other {# tasks}} fallaron",
+  "{hours, plural, =1 {# hour} other {# hours}}": "{hours, plural, =1 {# hour} other {# hours}}",
+  "{interfaceName} must start with \"{prefix}\" followed by an unique number": "{interfaceName} debe comenzar con \"{prefix}\" seguido de un número único",
+  "{key} Key": "Clave {key}",
+  "{license} contract, expires {date}": "Contrato de {license}, vence el {date}",
+  "{minutes, plural, =1 {# minute} other {# minutes}}": "{minutes, plural, =1 {# minute} other {# minutes}}",
+  "{n, plural, =0 {No Errors} one {# Error} other {# Errors}}": "{n, plural, =0 {No Errors} one {# Error} other {# Errors}}",
+  "{n, plural, =0 {No Tasks} one {# Task} other {# Tasks}} Configured": "{n, plural, =0 {No Tasks} one {# Task} other {# Tasks}} configurada",
+  "{n, plural, =0 {No errors} one {# Error} other {# Errors}}": "{n, plural, =0 {No errors} one {# Error} other {# Errors}}",
+  "{n, plural, =0 {No keys} =1 {# key} other {# keys}}": "{n, plural, =0 {No keys} =1 {# key} other {# keys}}",
+  "{n, plural, =0 {No open files} one {# open file} other {# open files}}": "{n, plural, =0 {No open files} one {# open file} other {# open files}}",
+  "{n, plural, one {# CPU} other {# CPUs}}": "{n, plural, one {# CPU} other {# CPUs}}",
+  "{n, plural, one {# Environment Variable} other {# Environment Variables} }": "{n, plural, one {# Environment Variable} other {# Environment Variables} }",
+  "{n, plural, one {# GPU} other {# GPUs}} isolated": "{n, plural, one {# GPU} other {# GPUs}} aislada",
+  "{n, plural, one {# boot environment} other {# boot environments}} has been deleted.": "{n, plural, one {# boot environment} other {# boot environments}} fue eliminado.",
+  "{n, plural, one {# core} other {# cores}}": "{n, plural, one {# core} other {# cores}}",
+  "{n, plural, one {# docker image} other {# docker images}} has been deleted.": "{n, plural, one {# docker image} other {# docker images}} fue eliminada.",
+  "{n, plural, one {# thread} other {# threads}}": "{n, plural, one {# thread} other {# threads}}",
+  "{n, plural, one {Failed Disk} other {Failed Disks} }": "{n, plural, one {Failed Disk} other {Failed Disks} }",
```

Comment:
So those strings specify how to translate plural words based on your language.
Here `Failed Disk` and `Failed Disks` are translatable.
`{n, plural, one {Failed Disk} other {Failed Disks} }` may become something like `{n, plural, one {Disco averiado} other {Discos averiados} }`

See [TRANSLATING.md](https://github.com/truenas/webui/blob/master/TRANSLATING.md) and [online editor](http://format-message.github.io/icu-message-format-for-translators/editor.html) for help.